lemur/certificates/schemas.py

```diff
@@ -146,7 +146,9 @@ def load_data(self, data):
                 data["extensions"]["subAltNames"] = {"names": []}
             elif not data["extensions"]["subAltNames"].get("names"):
                 data["extensions"]["subAltNames"]["names"] = []
-            data["extensions"]["subAltNames"]["names"] += csr_sans
+
+            if not data["extensions"]["subAltNames"]["names"]:
```

Comment:
is there a reason for this additional check?
do you think it is important to prevent overwriting ?
`data["extensions"]["subAltNames"]["names"]`

Reply:
This fix bug that I described here: https://github.com/Netflix/lemur/issues/2921.
When we always append `csr_sans` it leads to duplicates sans when cert is rotated and number of san records increase in a progression.

Reply:
sorry for this taking so long. I would suggest to change 
```
if not data["extensions"]["subAltNames"]["names"]:
   data["extensions"]["subAltNames"]["names"] += csr_sans
```
to
```
   data["extensions"]["subAltNames"]["names"] = csr_sans
```

Lemur does take the CSR provided, for the request to the CA, and hence, the SANS provided in the CSR are considered. If the SANS provided via the UI do not match the CSR this would result into issues during the rotation, so I believe we should take the CSR as the ground truth